src/test/lib/MtuTest.cpp

```diff
@@ -49,12 +49,13 @@ struct ResetSettings {
 void
 QuicTestMtuSettings()
 {
+    MsQuicRegistration Registration(true);
```

Comment:
There are a number of changes in the test code that are unnecessary, and I assume accidental. Please remove them if you can.